gatsby-browser.js

```diff
@@ -1,10 +1,9 @@
+import './src/styles/fonts.css'
```

Comment:
Wouldn't be more performant to load these `fonts` within the `<head>`? As we currently make use of Google Fonts, Base Store could benefit from Google's built-in optimizations, such as:
* [Combination of requests](https://www.smashingmagazine.com/2019/06/optimizing-google-fonts-performance/#3-combine-requests);
* [DNS Preconnect](https://www.smashingmagazine.com/2019/06/optimizing-google-fonts-performance/#preconnect).

Reply:
stringifying this file content into the <head> you really improve performance! nice catch! However, we are not using google fonts now, we are using local ones, right?

Reply:
You're right: serving fonts in the same server is better than connecting to Google, right? Thanks!

Reply:
Afaik, defining the fonts families and the usage of these new fonts, increase performance, decreasing the recalculate styles that decrease the tbt, and can decrease the CLS if exists.